PopulateDatabase.py

```diff
@@ -16,10 +16,12 @@
 from core.models.MemberModels import Member, Staffer
 from core.models.QuizModels import Question, Answer
 from core.models.TransactionModels import Transaction
+from django.contrib.auth.models import Group
 from django.core.exceptions import ValidationError
 from django.db.utils import IntegrityError
 from django.utils.timezone import timedelta
 
+import buildPermissions
```

Comment:
"Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability." [pep8](http://pep8.org/#package-and-module-names)

Reply:
We haven't been too consistent with this before, but all new modules should follow this. We can refactor the rest later.